frontend/src/views/SettingWorkspaceMember.vue

```diff
@@ -2,6 +2,12 @@
   <div v-if="allowAddOrInvite" class="w-full flex justify-center mb-6">
     <MemberAddOrInvite />
   </div>
+  <FeatureAttention
+      v-if="!hasAdminFeature"
```

Comment:
Could you also please change this to hasRbacFeature in all the places?

Reply:
Done